modules/openxr/extensions/openxr_extension_wrapper.h

```diff
@@ -43,18 +43,12 @@ class OpenXRAPI;
 class OpenXRActionMap;
 
 class OpenXRExtensionWrapper {
-protected:
-	OpenXRAPI *openxr_api = nullptr;
-
-	// Store extension we require.
-	// If bool pointer is a nullptr this means this extension is mandatory and initialisation will fail if it is not available
-	// If bool pointer is set, value will be set to true or false depending on whether extension is available
-	HashMap<String, bool *> request_extensions;
-
 public:
-	virtual HashMap<String, bool *> get_request_extensions() {
-		return request_extensions;
-	}
```

Comment:
Just so I understand, why is this being removed from the `openxr_extension_wrapper` class?

Reply:
This is to remove member variables from the class so we can implement the class as a `GDExtension` as well. There is no point in storing these anyway so `get_request_extensions` is now simply a virtual function that has to be implemented on the subclass.